### PR TITLE
FAQ #international_phone_numbers: technical hotline times changed.

### PR DESCRIPTION
**Pull request by dsarkar**
EN
image

DE
image

---
Internal Tracking ID: [EXPOSUREAPP-12352](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12352)

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -1278,7 +1278,7 @@
                                 "textblock": [
                                     "You can reach us by phone from within Germany and from abroad. The national numbers are free of charge, but are only available from within Germany. The international numbers are <b>not free of charge</b> - the cost of the international call is based on the contract with your provider.",
                                     "<b>Technical Hotline</b>",
-                                    "The technical hotline will help you with technical questions about the Corona-Warn-App, for instance if you have problems with the exposure logging. It is available Monday to Saturday from 7 a.m. to 10 p.m. German local time (except on German national holidays):",
+                                    "The technical hotline will help you with technical questions about the Corona-Warn-App, for instance if you have problems with the exposure logging. It is available Monday to Friday from 9 a.m. to 17 p.m. German local time (except on German national holidays):",
```
Comment (Ein-Tim) [inline src/data/faq.json:1281]: I think it makes sense to add a note that the hotline is only available in German.

Comment (dsarkar) [reply src/data/faq.json]: ok, good idea!

File: src/data/faq.json [hunk 2]
```diff
@@ -1278,7 +1278,7 @@
                                 "textblock": [
                                     "You can reach us by phone from within Germany and from abroad. The national numbers are free of charge, but are only available from within Germany. The international numbers are <b>not free of charge</b> - the cost of the international call is based on the contract with your provider.",
                                     "<b>Technical Hotline</b>",
-                                    "The technical hotline will help you with technical questions about the Corona-Warn-App, for instance if you have problems with the exposure logging. It is available Monday to Saturday from 7 a.m. to 10 p.m. German local time (except on German national holidays):",
+                                    "The technical hotline will help you with technical questions about the Corona-Warn-App, for instance if you have problems with the exposure logging (only available in German). It is available Monday to Friday from 9 a.m. to 17 p.m. German local time (except on German national holidays):",
```
Comment (MikeMcC399) [inline src/data/faq.json:1281]:
```suggestion
                                    "The technical hotline will help you with technical questions about the Corona-Warn-App, for instance if you have problems with the exposure logging (only available in German). It is available Monday to Friday from 9 a.m. to 5 p.m. German local time (except on German national holidays):",
```

a.m. / p.m. assumes a 12-hour clock, so the maximum is 12. 
17 p.m. is invalid

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
wrong time format in English

File: src/data/faq.json (hunk 2 above)
Comment (dsarkar) [reply src/data/faq.json:1281]: ha, true...

**Comment by MikeMcC399** [comment]
@dsarkar 

If the technical hotline will only be available in German, then also:

- https://www.coronawarn.app/en/faq/results/#available_languages
"The technical hotline ([0800 7540001](tel:08007540001)) answers in Turkish, English and German. More languages are to come. We'll keep you up to date."
- https://www.coronawarn.app/de/faq/results/#available_languages
"Die technische Hotline ([0800 7540001](tel:08007540001)) wird auf Deutsch, Englisch und Türkisch bedient. Weitere Sprachen sind geplant. Wir halten Sie auf dem Laufenden."

need to be changed.

**Comment by MikeMcC399** [comment]
The app and the FAQs are now out of sync. See also issue https://github.com/corona-warn-app/cwa-hotline/issues/23

The CWA Android 2.22.3 app and the [CWA website](https://www.coronawarn.app/en/) give conflicting information about the availability of the hotlines, so it currently unclear which information is correct.

Technical hotline | App information | Website information
--- | --- | ---
Availability | Mo - Fr 9 am to 5 pm | Mo - Sa 7 am to 10 pm
Language | German | Turkish, English and German

**Pull request merged**